src/commands/new.ts

```diff
@@ -30,11 +31,20 @@ export default {
     const cli = expo ? "expo-cli" : "react-native-cli"
     const bowserPath = path(`${meta.src}`, "..")
     const boilerplatePath = path(bowserPath, "boilerplate")
+    const boilerplateCopy = path(tempy.directory(), "boilerplate")
+    if (expo) {
+      filesystem.copy(boilerplatePath, boilerplateCopy)
+      const packageJsonPath = path(boilerplateCopy, "package.json")
+      const packageJson = filesystem.read(packageJsonPath, "json")
+      delete packageJson.scripts.prepare
```

Comment:
You won't need to do this based on our Slack conversation, but just FYI, there's a very cool way to handle this built-in to Gluegun!

```js
await toolbox.patching.update('package.json', pkgJson => {
  delete pkgJson.scripts.prepare
  return pkgJson
})
```

Reply:
Cool!